### PR TITLE
docs: Update WinUI 3 related article with the new discord link

### PR DESCRIPTION
**Pull request by agneszitte**
Update WinUI 3 related article with the new discord link

### CODE REVIEW

**Comment by agneszitte** [comment]
@mergify backport release/stable/5.1

File: doc/articles/uwp-vs-winui3.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ The questions below should help guide you to the best choice when creating a new
 
 ### Do you use 3rd-party libraries?
 
-While 3rd party vendors have offerings for both UWP and WinUI, UWP has been in the market longer. From Uno Platform perspective either will work, or if not, contact us at our [Discord channel](https://discord.com/invite/eBHZSKG).
+While 3rd party vendors have offerings for both UWP and WinUI, UWP has been in the market longer. From Uno Platform perspective either will work, or if not, contact us at our [Discord channel](https://platform.uno/discord).
```
Comment (morning4coffe-dev) [inline doc/articles/uwp-vs-winui3.md:42]: Thanks, @agneszitte. I somehow missed this 🤔 Just for a bit of standardization, I suggest  
```suggestion
While 3rd party vendors have offerings for both UWP and WinUI, UWP has been in the market longer. From Uno Platform perspective either will work, or if not, contact us on our [Discord Server](https://platform.uno/discord).
```

Comment (agneszitte) [reply doc/articles/uwp-vs-winui3.md]: thanks @morning4coffe-dev !

**Comment by agneszitte** [comment]
@mergify backport release/stable/5.1

**Pull request merged**